modules/setting/setting.go

```diff
@@ -580,17 +582,17 @@ func NewContext() {
 	AppURL = strings.TrimRight(AppURL, "/") + "/"
 
 	// Check if has app suburl.
-	url, err := url.Parse(AppURL)
+	appURL, err := url.Parse(AppURL)
```

Comment:
Changed `url` to `appURL` because `url` collides with the Go `url` package, and also to clarify the variable for use down below